e2e-tests/cypress/integration/project-application-documents.feature

```diff
@@ -186,3 +186,41 @@ Feature: Project Application documents page
         Given I navigate to "Ho Ho Hooo" project Overview page
         When I click on "Project application documents" link
         Then I verify that no documents found text displayed on the page
+
+    Scenario: search returns matching documents
+        Given I navigate to "Hinkley Point C New Nuclear Power Station Material Change 1" project Overview page
```

Comment:
Would be better to use the Cleve Hill Solar Farm project which was added specifically because it has an export of production document data so is more realistic.  For one thing it has proper file size values 😁

Reply:
changed project as suggested